### PR TITLE
Quick Help for OSCClient

### PR DESCRIPTION
**Pull request by gwsounddsg**
Adds Markup comments on `OSCClient` for the purpose of [quick help](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/SymbolDocumentation.html#//apple_ref/doc/uid/TP40016497-CH51-SW1) along with documenting the code for local reasoning.

### CODE REVIEW

**Comment by gwsounddsg** [comment]
Here's just the first pass at doing the quick help. Once we find what we like, I can take a larger chunk of the code to be reviewed.

File: Framework/SwiftOSC/Communication/OSCClient.swift [hunk 1]
```diff
@@ -1,34 +1,69 @@
...
+    
+    /**
+     *Send a message to current address and port*
+     
+    - Parameters:
+       - element: An OSCElement is an abstract protocol inherited by OSCMessage and OSCBundle.
+     
+    - Important:
+        If the OSCClient is not connected to a valid address and port, this method will not throw an error. You are responsible for managing
+        a vaild connection.
```
Comment (devinroth) [inline Framework/SwiftOSC/Communication/OSCClient.swift]: This is an good point.

**Comment by devinroth** [comment]
Everything looks fine to me. I say go ahead with whatever you want to tackle. I'll edit and add stuff as I see fit.

**Comment by gwsounddsg** [comment]
Fantastic. It might take me a couple of days. But I'll put it all in one pull request as opposed to a bunch of them.

**Pull request merged**